src/reference/03-Developers-Guide/00.md

```diff
@@ -2,12 +2,93 @@
 out: Developers-Guide.html
 ---
 
-Developer's Guide
------------------
-
-This is the set of documentation about the Architecture of sbt. This
-covers all the core components of sbt as well as the general notion of
-how they all work together. This documentation is suitable for those who
-wish to have a deeper understanding of sbt's core, but already
-understand the fundamentals of `Setting[_]`, `Task[_]` and constructing
-builds.
+  [pickling]: https://github.com/scala/pickling
+  [HowCanIHelp]: http://www.scala-sbt.org/community.html#how-can-I-help
+  [Sbt-Launcher]: Sbt-Launcher.html
+
+Developer's Guide (Work in progress)
+------------------------------------
+
+This is the set of documentation about the Architecture of sbt.
```

Comment:
MOdularization != Architecture.   Modularization is more of a "core principle" or "goal".  Archtiecture is the actual structure of each module.